backend/gamification/gamification.go

```diff
@@ -56,6 +55,15 @@ func PointCalculation(gs GameState, scanResults []checks.Check) (GameState, erro
 		}
 		gs.PointsHistory = append(gs.PointsHistory, gs.Points)
 	}
+	// Close the database
+	logger.Log.Info("Closing database")
```

Comment:
I would make it a debug log, to keep info logging as clean as possible

Reply:
I agree. It was however also in other parts of the code, so lets in the future go over all logging for consistency

Reply:
yes that is something that needs to be done, good point